### ISSUE TITLE
Allow more flexible notes, similar to Resources module

### ISSUE THREAD

**Issue by remocrevo** [coral-erm/coral#33]
All modules should provide the same notes features as the Resources module has. The notes code and database tables should be shared among all modules.

**Comment by benheet** [comment]
I agree that the notes feature within Resources would be beneficial within any module.

**Comment by scottvieira** [comment]
I have been wanting notes for the licensing module for a good time now.  I suspect once I start using the usage statistics module that I will want them there too.

**Comment by techsvcslib** [comment]
Note that in the Usage Statistics module, there is a field that I heavily use, called Interface Notes. Whatever change we make for that module, we need to keep this existing note field in mind. It’s where we store things like, where to navigate in each vendor’s admin UI to get to usage reports, as well as many other things.

Steve

**Comment by PaulPoulain** [comment]
In the long term, I think it would be helpful to factorize the code of the "notes" management.
